web/portal/client/cypress/e2e/Extension/Extension.cy.js

```diff
@@ -0,0 +1,93 @@
+import ExtensionCollection from '../../fixtures/Extension/getCollection.json';
+import newExtension from '../../fixtures/Extension/post.json';
+import ExtensionItem from '../../fixtures/Extension/getItem.json';
+import editExtension from '../../fixtures/Extension/put.json';
+
+describe('in Extension', () => {
+  beforeEach(() => {
+    cy.prepareGenericPactInterceptors('Extension');
+    cy.before();
+
+    cy.contains('Extension').click();
+
+    cy.get('h3').should('contain', 'List of Extensiones');
+
+    cy.get('table').should('contain', ExtensionCollection.body[0].name);
+  });
+
+  ///////////////////////
+  // POST
+  ///////////////////////
+  it('add Extension', () => {
+    cy.usePactIntercept(
+      {
+        method: 'POST',
+        url: '**/api/client/extensions*',
+        response: newExtension.response,
+        matchingRules: newExtension.matchingRules,
+      },
+      'createExtension'
+    );
+
+    cy.get('[aria-label=Add]').click();
+
+    cy.fillTheForm(newExtension.request);
```

Comment:
fillTheForm arguments should be passed in the order they are printed in screen (You can copy it from form.tsx). For instance:
```
const { fld1, fld2 } = newExtension.request;
 cy.fillTheForm({
    fld1,
    fld2
});